hangman.rb

```diff
@@ -0,0 +1,194 @@
+require "Colorize"
+
+class Hangman
+
+  def initialize
+    # List of possible game words. They are randomly selected during initialize.
+    @word_list =  ["w h a l e", "f l a g", "d o g", "f o r e s t",
+    "b r a v e", "r o s e", "g i f t", "s t a y", "a x", "d o l p h i n"]
+
+    # ASCII representations of the hangman graphics
+    @hanger_start   = """
+    |     _________
+    |     |/      |
+    |     |
+    |     |
+    |     |
+    |     |
+    |     |
+    | ____|___
+    """
+    @hanger_head = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |
+    |     |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+   @hanger_body = """
+
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |       |
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+   """.colorize(:red)
+
+    @hanger_arm1 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |       |\\
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_arm2 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_leg1 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |      /
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_leg2 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |      / \\
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @winner = """
+    HURAHH!!!! I'm alive!
+           (∞)
+           \\|/
+            |
+           / \\
+    """.colorize(:light_blue)
+
+    # This creates an array of possible ASCII outcomes as the user makes mistakes or wins
+    @hanger_list = [@hanger_head, @hanger_body, @hanger_arm1, @hanger_arm2, @hanger_leg1, @hanger_leg2, @winner]

```

Comment:
Since creating the `@hanger_list` instance variable is a distinct process separate from all of the other initialization features, I'd consider putting it in a separate method so that the initialize method is more straightforward and easy to read.